src/utilities/graphql/transform.ts

```diff
@@ -10,9 +10,20 @@ const TYPENAME_FIELD: FieldNode = {
 };
 
 /**
- * Adds `__typename` to all selection sets in the document.
+ * Adds `__typename` to all selection sets in the document. The operation
+ * definition's selection set remains unchanged.
```

Comment:
```suggestion
 * Adds `__typename` to all selection sets in the document. 
 * Returns a modified copy - the original document is not mutated.
```

Or did you mean this?